admin/python/lsst/qserv/admin/nodeAdmin.py

```diff
@@ -84,7 +84,7 @@ class _SSHCommand(object):
     # - known_hosts file has tendency to become inconsistent, ignore it
     sshOptions = ['StrictHostKeyChecking=no']
 
-    def __init__(self, host, cwd=None):
+    def __init__(self, host, cwd=None, kerberos=False, user=None):
```

Comment:
Note also that whole `_SSHCommand` class will disappear from here in the future, so you should copy it to a different location if you want to use it.